lib/svg4everybody.js

```diff
@@ -172,15 +175,15 @@ function svg4everybody(rawopts) {
 
 							// add the svg and id as an item to the xhr embeds list
 							xhr._embeds.push({
-								svg: svg,
+								node: parent,
 								id: id
 							});
 
 							// prepare the xhr ready state change event
 							loadreadystatechange(xhr);
 						} else {
 							// embed the local id into the svg
-							embed(svg, document.getElementById(id));
+							embed(parent, document.getElementById(id));
```

Comment:
Minor: would it be better to pass in both the `svg` and `parent` so that we don't have to search through the DOM again? Its likely that svg and use are close together, but its would be more efficient this way. You would have to change the obj being passed into `xhr._embeds` as well.